src/mame/atari/quantum.cpp

```diff
@@ -375,16 +376,17 @@ ROM_END
 
 ROM_START( quantum1 )
 	ROM_REGION( 0x014000, "maincpu", 0 )
-	ROM_LOAD16_BYTE( "136016.101",   0x000000, 0x002000, CRC(5af0bd5b) SHA1(f6e46fbebbf52294e78ae240fe2628c6b29b8dea) )
-	ROM_LOAD16_BYTE( "136016.106",   0x000001, 0x002000, CRC(f9724666) SHA1(1bb073135029c92bef9afc9ccd910e0ab3302c8a) )
-	ROM_LOAD16_BYTE( "136016.102",   0x004000, 0x002000, CRC(408d34f4) SHA1(9a30debd1240b9c103134701943c94d6b48b926d) )
-	ROM_LOAD16_BYTE( "136016.107",   0x004001, 0x002000, CRC(63154484) SHA1(c098cdbc339c9ea291c4c4fb203c60b3284e894a) )
-	ROM_LOAD16_BYTE( "136016.103",   0x008000, 0x002000, CRC(948f228b) SHA1(878ac96173a793997cc88be469ec1ccdf833a7e8) )
-	ROM_LOAD16_BYTE( "136016.108",   0x008001, 0x002000, CRC(e4c48e4e) SHA1(caaf9d20741fcb961d590b634250a44a166cc33a) )
-	ROM_LOAD16_BYTE( "136016.104",   0x00C000, 0x002000, CRC(bf271e5c) SHA1(012edb947f1437932b9283e49d025a7794c45669) )
-	ROM_LOAD16_BYTE( "136016.109",   0x00C001, 0x002000, CRC(d2894424) SHA1(5390025136b677b66d948c8cf6ea5e20203a4bae) )
-	ROM_LOAD16_BYTE( "136016.105",   0x010000, 0x002000, CRC(13ec512c) SHA1(22a0395135b83ba47eacb5129f34fc97aa1b70a1) )
-	ROM_LOAD16_BYTE( "136016.110",   0x010001, 0x002000, CRC(acb50363) SHA1(9efa9ca88efdd2d5e212bd537903892b67b4fe53) )
+	ROM_LOAD16_BYTE( "136016-101.2e",    0x000000, 0x2000, CRC(7e7be63a) SHA1(11b2d0168cdbaa7a48656b77abc0bcbe9408fe84) )
+	ROM_LOAD16_BYTE( "136016-106.3e",    0x000001, 0x2000, CRC(2d8f5759) SHA1(54b0388ef44b5d34e621b48b465566aa16887e8f) )
+	ROM_LOAD16_BYTE( "136016-102.2f",    0x004000, 0x2000, CRC(408d34f4) SHA1(9a30debd1240b9c103134701943c94d6b48b926d) )
+	ROM_LOAD16_BYTE( "136016-107.3f",    0x004001, 0x2000, CRC(63154484) SHA1(c098cdbc339c9ea291c4c4fb203c60b3284e894a) )
+	ROM_LOAD16_BYTE( "136016-103.2hj",   0x008000, 0x2000, CRC(bdc52fad) SHA1(c8ede54a4f7f555adffa5b4bfea6bf646a0d02d4) )
+	ROM_LOAD16_BYTE( "136016-108.3hj",   0x008001, 0x2000, CRC(dab4066b) SHA1(dbb82df8e6de4e0f9f6e7ddd5f07618864fce8f9) )
+	ROM_LOAD16_BYTE( "136016-104.2k",    0x00C000, 0x2000, CRC(bf271e5c) SHA1(012edb947f1437932b9283e49d025a7794c45669) )
+	ROM_LOAD16_BYTE( "136016-109.3k",    0x00C001, 0x2000, CRC(d2894424) SHA1(5390025136b677b66d948c8cf6ea5e20203a4bae) )
+	ROM_LOAD16_BYTE( "136016-105.2l",    0x010000, 0x2000, CRC(13ec512c) SHA1(22a0395135b83ba47eacb5129f34fc97aa1b70a1) )
+	ROM_LOAD16_BYTE( "136016-110.3l",    0x010001, 0x2000, CRC(acb50363) SHA1(9efa9ca88efdd2d5e212bd537903892b67b4fe53) )
+
```

Comment:
It looks like you’ve replaced the CRC/SHA1 values here with the ones from `quantum`.